templates/part.board.headerControls.php

```diff
@@ -13,19 +13,30 @@
     </form>
 </div>
 
-<button ng-if="params.filter!='archive'" ng-click="switchFilter('archive')" title="<?php p($l->t('Show archived cards')); ?>">
-    <i class="icon icon-archive" style="opacity:0.5;"></i>
-    <span class="hidden-visually"><?php p($l->t('Show archived cards')); ?></span>
-</button>
-<button ng-if="params.filter=='archive'" ng-click="switchFilter('')" title="<?php p($l->t('Hide archived cards')); ?>">
-    <i class="icon icon-archive"></i>
-    <span class="hidden-visually"><?php p($l->t('Hide archived cards')); ?></span>
-</button>
-<button ng-click="toggleCompactMode()" title="<?php p($l->t('Toggle compact mode')); ?>">
-	<i class="icon" ng-class="{ 'icon-toggle-compact-collapsed': compactMode, 'icon-toggle-compact-expanded': !compactMode }"></i>
-	<span class="hidden-visually"><?php p($l->t('Toggle compact mode')); ?></span>
-</button>
-<button ui-sref="board.detail({ id: id, tab: 0})"  title="<?php p($l->t('Show board details')); ?>">
-    <i class="icon icon-settings"></i>
-    <span class="hidden-visually"><?php p($l->t('Show board details')); ?></span>
-</button>
+<div class="section-label" data-toggle="tooltip" data-placement="bottom" title="<?php p($l->t('Show archived cards')); ?>">
```

Comment:
There should not be a wrapping div, you can also add the `data-toggle` and `data-placement` attributes directly to the `<button>` elements